warehouse/accounts/forms.py

```diff
@@ -191,7 +191,7 @@ def validate_password(self, field):
             except TooManyFailedLogins as err:
                 raise wtforms.validators.ValidationError(
                     _(
-                        "There have been too many unsuccessful login attempts. "
+                        f"There have been too many unsuccessful login attempts. "
```

Comment:
This f-string has no effect, as far as I can tell? It might actually break the code below, since it'll format `{time}` instead of letting the i18n machinery take care of it.